README.md

```diff
@@ -81,9 +81,17 @@ drupal self-update
 You can ask for support at Drupal Console gitter chat room [http://bit.ly/console-support](http://bit.ly/console-support).
 
 ## Getting The Project To Contribute
-
 For information about how to run this project for development follow instructions at [setup instructions](https://gist.github.com/jmolivas/97bbd07f328217be3564a434c5bd2618).
 
+An optional way you can execute this:
+```
+composer require --dev drupal/console:dev-master
+```
+You might have to execute too:
+```
+composer require symfony/dom-crawler:2.8 symfony:debug:2.8
```

Comment:
**well, firstly, master development branch already has ~2.8 versions of dom-crawler & debug as requirements.**
so, the statement `You might have to execute too` is only intended for the stable version (beta5) ....

secondly, regarding to `beta5` (which has 2.7 versions) i had no problems ...